### ISSUE TITLE
Upgrading BigBlueButton dependencies, especially those reaching EOL

### ISSUE THREAD

**Issue by solkh** [bigbluebutton/bigbluebutton#16097]
Dependecies used in BigBlueButton are very old, many of witch are reaching EOL soon:
- NGINX - 1.18 - EOL
- Tomcat 9 - released 9 yrs ago - EOL can be expected next yr
- Node js 16 - Active support: EOL, security support ends in 9 months 
- Redis Server - EOL
- Ubuntu 22.04 LTS already released (but not used!)

I think updating these dependecies would be relatively easy and very beneficial in lowering maintainance efforts in production environments

**Comment by kepstin** [comment]
Regarding Nginx and Redis, we're using the versions shipped with the Ubuntu LTS release. They are continuing to receive security updates maintained by Ubuntu, and we'd like to benefit from that work rather than maintain the packages ourselves. We will get new versions of those components when we switch to a newer Ubuntu version.

**Comment by solkh** [comment]
Is there any roadmap for switching to Ubuntu 22.04? Can we expect these upgrades with BBB v2.6 ?

**Comment by antobinary** [comment]
> Is there any roadmap for switching to Ubuntu 22.04? Can we expect these upgrades with BBB v2.6 ?

We have started looking at upgrading already. This would be happening in BBB 2.7 at the earliest. I just created an issue to track this https://github.com/bigbluebutton/bigbluebutton/issues/16100

Regarding Node -- bbb-html5 actually uses Node js 14 (still a few months of support) and our hands are a bit tied as we rely on Meteor.js  just created an issue to track this https://github.com/bigbluebutton/bigbluebutton/issues/16101
Several different components are more flexible and are on Node 16 currently. https://github.com/nodejs/release#release-schedule

Upgrading tomcat: https://github.com/bigbluebutton/bigbluebutton/issues/16102 -- BBB 2.5 and 2.6 will remain on 9.0.x, but for BBB 2.7 we'll shoot for Tomcat 11 or at least 10.1.x
